practice/src/css/sections/package.css

```diff
@@ -0,0 +1,111 @@
+.wrapper-package {
+  padding-bottom: 50px;
+}
+  
+.package-card-item {
+  position: relative;
+
+  &:hover {
+    border-radius: var(--primary-border-radius);
+    border: 4px solid var(--primary-border-radius-color);
+    transition: transform 0.3s ease;
+    transform: scale(0.98);
+    padding: 5px;
+  }
+}
+
+.package-card-item-image {
+  border-radius: var(--primary-border-radius);
+  width: 100%;
+  height: 380px;
+}
+  
+.package-information {
+  position: absolute;
+  bottom: 35px;
+  left: 40px;
+}
+  
+.package-icon {
+  padding-bottom: 5px;
+}
+  
+.package-address {
+  display: flex;
+  gap: 4px;
+  align-items: center;
+}
+  
+.package-introduction {
+  padding-top: 30px;
+  padding-bottom: 30px;
+}
+  
+.package-evaluation {
+  display: flex;
+  gap: 10px;
+  padding-top: 30px;
+  padding-bottom: 25px;
+}
+  
+.introduction-desc {
+  padding-top: 10px;
+}
+  
+.package-desc {
+  color: var(--gray-text-color);
+}
+
+.package-title {
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-medium);
+  line-height: var(--secondary-line-height);
+  color: var(--primary-text-color);
+  display: flex;
+  gap: 10px;
+}
+
+.package-card-desc {
+  font-weight: var(--font-weight-regular);
+  font-size: var(--font-size-small);
+  line-height: var(--tertiary-line-height);
+  color: var(--dark-grey-text-color);
+}
+
+.evaluation-title {
+  color: var(--dark-text-color);
+  font-weight: var(--font-weight-semibold);
+  font-size: var(--font-size-lg);
+  line-height: var(--primary-line-height);
+}
+
+.btn-package:hover {
+  color: var(--tertiary-text-color);
+  background-color: var(--primary-background-color);
+}
+
+.package-card {
+  display: grid;
+  gap: 20px;
+}
+  
+/* Tablet */
+@media (min-width: 768px) {
+  .package-card {
+    grid-template-columns: repeat(auto-fit, minmax(200px, auto));
+  }
+}
+  
+/* Desktop */
+@media (min-width: 1440px) {
+  .package-introduction {
```

Comment:
@JennyDoJD I am sure why we need `width: 410px`

Reply:
I want introduction-desc break down the line like design